BlueprintUICommonControls/Sources/AccessibilityContainer.swift

```diff
@@ -61,7 +61,9 @@ extension AccessibilityContainer {
 extension UIView {
     func recursiveAccessibleSubviews() -> [Any] {
         subviews.flatMap { subview -> [Any] in
-            if let accessibilityElements = subview.accessibilityElements {
+            if subview.accessibilityElementsHidden {
```

Comment:
```suggestion
            if subview.accessibilityElementsHidden || subview.isHidden {
```

I'm thinking we should check this too

Reply:
`isHidden` doesn't normally block subviews from accessibility, does it?

Reply:
Hmm I thought it did! We should definitely match the system

Reply:
IMO I would _expect_ it do (since they're otherwise completely visible from most users). To not definitely feels like a foot gun

Reply:
(Does `accessibilityElementsHidden` default to `isHidden`?)

Reply:
I agree with KB that we should match the system, so I checked: it looks like `isHidden` does normally suppress nested a11y elements.

Reply:
This method is also in listable, I'll open a PR to add it there once this lands.

Reply:
So we _do_ want to add the `|| subview.isHidden` check to this?

- [ ] Add `isHidden` check

Reply:
> So we _do_ want to add the `|| subview.isHidden` check to this?

Yeah, I think so.